src/sbl_filing_api/routers/filing.py

```diff
@@ -352,7 +352,7 @@ async def get_latest_submission_report(request: Request, lei: str, period_code:
 @requires("authenticated")
 async def get_submission_report(request: Request, response: Response, lei: str, period_code: str, id: int):
     sub = await repo.get_submission(request.state.db_session, id)
-    if sub:
+    if sub and not sub.state == SubmissionState.VALIDATION_IN_PROGRESS:
```

Comment:
This won't work.  Submissions could go to VALIDATION_ERROR without writing a report, or SUBMISSION_FAILED, SUBMISSION_UPLOAD, etc.  So check for completed states